src/safeposix/syscalls/fs_calls.rs

```diff
@@ -2001,6 +2001,43 @@ impl Cage {
         0 //getcwd has succeeded!;
     }
 
+
+    //------------------SHMHELPERS----------------------
+
+    pub fn rev_shm_find_index_by_addr(rev_shm: &Vec<(u32, i32)>, shmaddr: u32) -> Option<usize> {
+        for (index, val) in rev_shm.iter().enumerate() {
+            if val.0 == shmaddr as u32 {
+                return Some(index);
+            }
+        }
+        None
+    }
+
+    pub fn rev_shm_find_addrs_by_shmid(rev_shm: &Vec<(u32, i32)>, shmid: i32) -> Vec<u32> {
+
+        let mut addrvec = Vec::new();
+        for (_index, val) in rev_shm.iter().enumerate() {
```

Comment:
Since `_index` is unused, is it possible to just delete it and do `for val in rev_shm.iter()`. Same applies to `search_for_addr_in_region` function.